src/main/java/frc/robot/subsystems/HoodSubsystem.java

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.revrobotics.CANSparkMax.IdleMode;
+import edu.wpi.first.wpilibj.XboxController;
+
+
+public class HoodSubsystem extends SubsystemBase {
+
+  /** Creates a new ExampleSubsystem. */
+  private static final int deviceID = 1;
```

Comment:
See the existing constant's file. Constants such as CAN ids and PID values should be store there as opposed to right in the file.